sale_cart_quotation/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Glue module between sale_quotation and sale_cart
```

Comment:
This module is not a glue module only. It adds the functionality of requesting a quotation for a cart. This should be specified into the description.